### ISSUE TITLE
Hello author, may I ask if there is support for tools similar to Postbot AI in Postman in the future

### ISSUE THREAD

**Issue by cfy3133** [usebruno/bruno#2203]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

AI interface testing

### Mockups or Images of the feature

![1714440522622](https://github.com/usebruno/bruno/assets/43978367/8a01d920-625d-4c5c-b829-e23acebe71e2)

**Comment by princebillygk** [comment]
Yes I use this feature most of the time in postman will be great if the author adds this feature to roadmap

**Comment by james-ha-bruno** [comment]
Hey all, just checking to see if you've tried our recent AI options in v4? 
- https://docs.usebruno.com/ai/bruno-ai/introduction

@princebillygk @cfy3133
